core/src/test/java/com/google/android/fhir/index/ResourceIndexerTest.kt

```diff
@@ -431,5 +506,56 @@ class ResourceIndexerTest {
 
     // See: https://www.hl7.org/fhir/valueset-currencies.html
     const val FHIR_CURRENCY_SYSTEM = "urn:iso:std:iso:4217"
+
+    val TEST_OBSERVATION_EFFECTIVE_PERIOD = Observation()
+
+    init {
+      TEST_OBSERVATION_EFFECTIVE_PERIOD.apply {
+        id = "non-null ID"
+        effective =
+          Period().apply {
+            start = Date(1516055094)
```

Comment:
Can we add the values in any human readable form? E.g.:
```suggestion
            start = Date(1516055094) // Tue Jan 16 03:54:54 IST 2018
```